addons/core/defines.hpp

```diff
@@ -13,6 +13,7 @@
 #define RADIO_OWNER 7
 #define SPEAKER_OFFSET 8
 #define POWER_OFFSET 9
+#define HALFDUPLEX_OVERRIDE_OFFSET 10
```

Comment:
Happy to change it to a different offset, I was already figuring this would be a problem. Does ArmA just allow one to access 0-9 and suddenly e.g. 30 (with possibly nothing inbetween)?

Or is there another way for me to properly store the setting?